src/pages/index.js

```diff
@@ -39,6 +39,18 @@ const popupContentGatsby = `
 
 const IndexPage = () => {
   const markerRef = useRef();
+  const [location,setLocation] = useState(LOCATION);
+  const [center,setCenter] = useState(CENTER);
+
+  useEffect(()=>{
+    navigator.geolocation.getCurrentPosition(
```

Comment:
what do you think about moving this handler into the `mapEffect` function? that's essentially doing the same thing, it's just triggered upon a `useEffect` inside of the `Map` component. mostly because we already have similar logic in there already